src/components/AddComponent/AddComponentForm.tsx

```diff
@@ -5,10 +5,30 @@ import isEmpty from 'lodash/isEmpty';
 import { FormFooter } from '../../shared';
 import { useFormValues } from '../form-context';
 import { useWizardContext } from '../Wizard/Wizard';
+import { GitOptions } from './GitOptions';
 import { SourceField } from './SourceField';
 
 export type AddComponentValues = {
   source: string;
+  git: {
+    reference: string;
+    contextDir: string;
+  };
+  detectedComponents?: {
```

Comment:
Is this based on the latest kube API specs for `ComponentDetectionQuery`? I think there is a component resource stub that has all the default values for component resource. Ref - https://docs.google.com/document/d/1CBeRA8LrdsDBG4AqFYqn4aovcd969lP_1-41OUAN2Bk/edit#heading=h.apl9rlcsyxg8

Reply:
Yes I had mocked the ComponentDetectionQuery data from that doc in `create-utils.ts`: https://github.com/openshift/hac-dev/pull/25/files#diff-18923f7bff33b78fd8fb3983b01068eb30ac83937137a1580f332ee7560e3361R87-R128